src/igvfd/schemas/publication.json

```diff
@@ -0,0 +1,135 @@
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/attachment"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "title": {
+            "title": "Title",
+            "description": "Title of the publication or communication.",
+            "uniqueKey": true,
+            "type": "string"
+        },
+        "abstract": {
+            "title": "Abstract",
+            "description": "Abstract of the publication or communication.",
+            "type": "string"
+        },
+        "authors": {
+            "title": "Authors",
+            "type": "string"
+        },
+        "date_published": {
+            "title": "Publication date",
+            "description": "The date the publication or communication was published; must be in YYYY-MM-DD format.",
+            "type": "string",
+            "oneOf": [
+                {
+                    "format": "date"
+                },
+                {
+                    "comment": "Historical date formats, not valid for new submissions.",
+                    "permission": "import_items",
+                    "pattern": "^[0-9]{4}[^-]*$"
+                }
+            ]
+        },
+        "date_revised": {
+            "title": "Date revised",
+            "type": "string",
+            "anyOf": [
+                {
+                    "format": "date-time"
+                },
+                {
+                    "format": "date"
+                }
+            ]
+        },
+        "issue": {
+            "title": "Issue",
+            "description": "The issue of the publication.",
+            "type": "string"
+        },
+        "page": {
+            "title": "Page",
+            "description": "Pagination of the reference",
+            "type": "string"
+        },
+        "volume": {
+            "title": "Volume",
+            "description": "The volume of the publication.",
+            "type": "string"
+        },
+        "journal": {
+            "title": "Journal",
+            "description": "The journal of the publication.",
+            "type": "string"
+        },
+        "identifiers": {
+            "title": "Identifiers",
+            "description": "The identifiers that reference data found in the object.",
+            "type": "array",
+            "uniqueItems": true,
+            "items": {
+                "title": "Identifier",
+                "description": "An identifier that references data found in the object.",
+                "type": "string",
+                "uniqueKey": "publication:identifier",
+                "pattern": "^(PMID:[0-9]+|doi:10\\.[0-9]{4}[\\d\\s\\S\\:\\.\\/]+|PMCID:PMC[0-9]+|[0-9]{4}\\.[0-9]{4})$"
```

Comment:
I think the pattern should support whatever identifiers are in BioRxiv (and maybe it is already, just make sure)

Reply:
BioRxiv uses doi, so it is supported. I put an insert to reflect this.